syntax = "proto3";

package util;

import "google/protobuf/timestamp.proto";

option java_package = "common.model.protoUtils";

message LocalTimestampProto {
  google.protobuf.Timestamp timestamp = 1;
  string time_zone = 2;
  //TODO: Need to decide how to model this: https://en.wikipedia.org/wiki/List_of_time_zone_abbreviations
  //https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html
}